tests/core/admin.py

```diff
@@ -20,12 +20,25 @@ def for_delete(self, row, instance):
         return self.fields['name'].clean(row) == ''
 
 
+class MBookResource(ModelResource):
```

Comment:
Please see note above about renaming this to 'SummaryBookResource' so that it is more like a real world example.

Reply:
OK, I added `BookSummaryResource` to the tests.

Reply:
I got a bit confused here.  I don't think we need a new model instance (correct me if I'm wrong).  My concern is that it might confuse people who are trying to understand the package by using the example application.

I think that you could use `BookNameResource` here so it matches up with the documentation you added.

```
class BookAdmin(ImportExportMixin, admin.ModelAdmin):
    list_display = ('name', 'author', 'added')
    list_filter = ['categories', 'author']
    resource_classes = [BookResource, BookNameResource]
```

What do you think?  Am I missing something?

Reply:
Originally I wanted to test both cases - the `resource_class` with single class and  with list.

Now with `resource_classes` I simplified the code, so the case with one class and multiple classes is not different (admin shows the field in both cases). So I added the second resource class to the original `BookAdmin` and use code from the docs example.

Reply:
Looks good - it would be cool if the dropdown is not shown if there is only one class in the list.  This will be the case for the majority of users I think, so it might be confusing if they see a dropdown with only 1 option.  Maybe in this case it can default to showing the resource class name only.

![2021-12-22_12-14](https://user-images.githubusercontent.com/6249838/147091278-ba5d049a-646f-4035-8fa7-9b6b95efad22.png)

Reply:
I don't know how to just show the resource class (without making some kind of output text widget). I can disable the field and add help_text:
![Snímek obrazovky_2021-12-22_15-09-21](https://user-images.githubusercontent.com/156755/147105740-9b05606c-91d0-4984-8881-8445b2e84201.png)

I can remove the field entirely, but I think it is good, that the new feature is visible even if not used.

Reply:
I agree on feature visibility, but I'd say that having it visible even when there is only one option should be done/forced using an option, so that the "Resource" row does not show at all if there is only one option. IMHO this makes the UI cleaner and is backwards compatible (no user surprise).

Reply:
I agree with @manelclos that it would be better not to show if there is only one option.

Reply:
OK, I am returning back to removing the field if there is only one resource.